src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -38,14 +38,16 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults(array(
             'choice_loader' => function (Options $options) {
                 $regions = $options['regions'];
+                $country = $options['country'];
 
-                return new CallbackChoiceLoader(function () use ($regions) {
-                    return self::getTimezones($regions);
+                return new CallbackChoiceLoader(function () use ($regions, $country) {
```

Comment:
`?string $country`?

Reply:
I can't do that, it seems not to work passing typehint to anonymous function.

Reply:
Typehints on an anonymous function works fine. Not typehinting variables inherited from parent scope through the `use` keyword (and would be useless) 😄